api/spec/v1.yaml

```diff
@@ -355,6 +357,36 @@ paths:
                 $ref: '#/components/schemas/ConsensusEventList'
         <<: *common_error_responses
 
+  /consensus/roothash_messages:
+    get:
+      parameters:
+        - *limit
+        - *offset
+        - in: query
+          name: runtime
```

Comment:
This is presumably a required parameter? (Edit: I see now that it is, except if `round` is omitted. We should mention this in arg descriptions.)

Reply:
huh maybe we should just make it required. in practice you can't get recent messages from all runtimes because there's no index for that

Reply:
yea I think postgres is not smart enough to scan n_runtimes indexes and merge the results. Let's make this required for now, and we can relax it later if we find a use case?

Reply:
ok requiring